prac_08/squaring.kv

```diff
@@ -0,0 +1,21 @@
+BoxLayout:
+    orientation: 'vertical'
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput:
+            id: input_number
+            text: '7'
+            font_size: 48
+            multiline: False
+        Button:
+            text: 'Square'
+            # the following line specifies the function in the app class to call when the button is pressed
+            on_press: app.handle_calculate(input_number.text)
+        Label:
+            id: output_label
+            font_size: 48
+            color: (7, 0, 1, 1)
+    BoxLayout:
```

Comment:
You don't need to add a BoxLayout if there is only one element in it

Reply:
🫡